src/OpenRiaServices.Client.HttpDomainClient/Framework/BinaryHttpDomainClientFactory.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+
+namespace OpenRiaServices.Client.HttpDomainClient
+{
+    public class BinaryHttpDomainClientFactory
+        : DomainClientFactory
+    {
+        private Func<HttpClient> httpClientFactory;
+
+        public BinaryHttpDomainClientFactory()
+            : this(new HttpClientHandler()
+            {
+                CookieContainer = new System.Net.CookieContainer(),
+                UseCookies = true,
+                AutomaticDecompression = System.Net.DecompressionMethods.Deflate | System.Net.DecompressionMethods.GZip,
+            })
+        {
+
+        }
+
+        public BinaryHttpDomainClientFactory(HttpMessageHandler messageHandler)
+            : this(() => new HttpClient(messageHandler, disposeHandler: false))
+        {
+        }
+
+        public BinaryHttpDomainClientFactory(Func<HttpClient> httpClientFactory)
```

Comment:
The intended usage is for folks on .net5 + who want to use the `ÌHttpClientFactory` or similar.
It might make sense to add a string parameter, such as the uri, interface name or DomainContext name ?

An alternative is to add a virtual method "CreateHttpClient(Type serviceContract, Uri serviceUri, bool requiresSecureEndpoint)"   which can easily be overridden to produce the same result